src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+public class Square extends Shape {
+    private double side;
+
+    public Square(String color, double side) {
+        super(Colors.valueOf(color));
+        this.side = Math.round(Math.abs(side));
+    }
+
+    public double getSide() {
+        return side;
+    }
+
+    public void setSide(double side) {
+        this.side = side;
```

Comment:
Consider validating the `side` value in the `setSide` method to ensure it is non-negative, similar to the constructor.